interfaces/builtin/fuse_support.go

```diff
@@ -72,11 +76,14 @@ deny /etc/fuse.conf r,
 `
 
 func init() {
+	// Ubuntu 14.04 does not support the fuse-support interface.
 	registerIface(&commonInterface{
 		name:                  "fuse-support",
 		summary:               fuseSupportSummary,
+		implicitOnCore:        true,
+		implicitOnClassic:     !(release.ReleaseInfo.ID == "ubuntu" && release.ReleaseInfo.VersionID == "14.04"),
```

Comment:
This looks like an unrelated change. I'm not saying it is wrong.... It seems that this sort of thing should be captured elsewhere. Eg, I can imagine Fedora or SUSE might have different interfaces. I mention this only because if going that direction, maybe it's best to not include this change in this PR.